src/components/CalculatorInput.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function CalculatorInput() {
+  return (
+    <input type="text" name="calc" className="input" />
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the calculator screen :clap: However it would be nice to use a tag that have **read only** property (`e.g. <p>, <div>, <span>, ...`) or add **readonly** property to the input to avoid the input value been change directly